dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,154 @@
+
+The Windows Forms [PrintDialog](../../../framework/winforms/controls/printdialog-component-windows-forms.md) component is a pre-configured dialog box used to select a printer, choose the pages to print, and determine other print-related settings in Windows-based applications. It's a simple solution for printer and print-related settings instead of configuring your own dialog box. You can enable users to print many parts of their documents: print all, print a selected page range, or print a selection. By relying on standard Windows dialog boxes, you create applications whose basic functionality is immediately familiar to users. The <xref:System.Windows.Forms.PrintDialog> component inherits from the <xref:System.Windows.Forms.CommonDialog> class.
+
+## Working with the Component
+
+Use the <xref:System.Windows.Forms.CommonDialog.ShowDialog%2A> method to display the dialog at run time. This component has properties that relate to either a single print job (<xref:System.Drawing.Printing.PrintDocument> class) or the settings of an individual printer (<xref:System.Drawing.Printing.PrinterSettings> class). One of the two, in turn, may be shared by multiple printers.
+
+ The show dialog box method helps you to add print dialog box to the form. The <xref:System.Windows.Forms.PrintDialog> component appears in the tray at the bottom of the Windows Forms Designer in Visual Studio.
+
+## How to: Capture User Input from a PrintDialog at Run Time
+
+You can set options related to printing at design time. Sometimes you may want to change these options at run time, most likely because of choices made by the user. You can capture user input for printing a document using the <xref:System.Windows.Forms.PrintDialog> and the <xref:System.Drawing.Printing.PrintDocument> components.  
+  
+### To change print options programmatically  
+  
+01. Add a <xref:System.Windows.Forms.PrintDialog> and a <xref:System.Drawing.Printing.PrintDocument> component to your form.  
+  
+01. Set the <xref:System.Windows.Forms.PrintDialog.Document%2A> property of the <xref:System.Windows.Forms.PrintDialog> to the <xref:System.Drawing.Printing.PrintDocument> added to the form.  
+  
+    ```vb  
+    PrintDialog1.Document = PrintDocument1  
+    ```  
+  
+    ```csharp  
+    printDialog1.Document = PrintDocument1;  
+    ```  
+  
+01. Display the <xref:System.Windows.Forms.PrintDialog> component by using the <xref:System.Windows.Forms.CommonDialog.ShowDialog%2A> method.  
+  
+    ```vb  
+    PrintDialog1.ShowDialog()  
+    ```  
+  
+    ```csharp  
+    printDialog1.ShowDialog();  
+    ```  
+  
+01. The user's printing choices from the dialog will be copied to the <xref:System.Drawing.Printing.PrinterSettings> property of the <xref:System.Drawing.Printing.PrintDocument> component.  
+
+## How to: Create Standard Windows Forms Print Jobs
```

Comment:
Fix the casing on these headers and simplify when you can. There are a few more to change in the article
```suggestion
## How to create a print job
```

Reply:
ok